### ISSUE TITLE
Graph explorer does not start, webpack missing

### ISSUE THREAD

**Issue by AmberKiso** [iron-fish/ironfish#906]
### What happened?

IronFish Version 0.1.19
Windows 10 Home
Current branch is master.

Following the instructions in ironfish-graph-explorer\README.md I get an error and the utility does not start:

> yarn start
> yarn run v1.22.17
> $ webpack serve
> 'webpack' is not recognized as an internal or external command,

I checked for existing issues, noticed #440, but this seems different. The issue seems to be that I don't have webpack installed.

### Version

0.1.19

### Relevant log output

```shell
C:\Users\mjolnir\Documents\GitHub\ironfish\ironfish-cli>yarn start:once chain:export
yarn run v1.22.17
$ yarn build && yarn start:js chain:export
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node bin/run chain:export
Exporting chain from 1 -> 71000 to C:\Users\mjolnir\Documents\GitHub\ironfish\ironfish-graph-explorer\src\data.json
Exporting blocks: [========================================] 71000/71000 100% | ETA: 0s
Export complete
Done in 93.08s.

C:\Users\mjolnir\Documents\GitHub\ironfish\ironfish-cli>
C:\Users\mjolnir\Documents\GitHub\ironfish\ironfish-cli>
C:\Users\mjolnir\Documents\GitHub\ironfish\ironfish-cli>
C:\Users\mjolnir\Documents\GitHub\ironfish\ironfish-cli>cd ..\ironfish-graph-explorer

C:\Users\mjolnir\Documents\GitHub\ironfish\ironfish-graph-explorer>yarn start
yarn run v1.22.17
$ webpack serve
'webpack' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by QustomQure** [comment]
npm install webpack
should help

**Comment by AmberKiso** [comment]
Thanks. Did that but still get the same result.
Also the update has quite a few vulnerabilities: 21 vulnerabilities (9 moderate, 12 high)

**Comment by AmberKiso** [comment]
Just needed to do a `yarn install` and it fetched the correct webpack, so a documentation change will fix this.

**Comment by AmberKiso** [comment]
Fixed by #1052
